src/skmultiflow/meta/learn_nse.py

```diff
@@ -265,11 +270,33 @@ def __vote_proba(self, X, t, classes):
                     h = self.ensemble[i]
                     y_predicts = h.predict_proba(X[m].reshape(1, -1))
                     y_predicts /= np.linalg.norm(y_predicts, ord=1, axis=1, keepdims=True)
-                    votes += self.ensemble_weights[i] * y_predicts
+                    try:
+                        votes += self.ensemble_weights[i] * y_predicts
+                    except ValueError:
+                        # sklearn learner
+                        if hasattr(h, 'classes_'):
+                            obs_classes = h.classes_
+                        else:  # skmultiflow learner
+                            obs_classes = h.classes
+                        votes += self.ensemble_weights[i] * \
```

Comment:
That is a valid concern. Do you have an idea of how to implement that? In other words, how to retrieve the classes observed by an arbitrary classifier?

Reply:
I am thinking on two options:

1. Raise a warning :-)
2. Make the `classes` parameter in `partial_fit` a required parameter

Reply:
I agree with both options. Particularly regarding option 2, I believe skmultiflow already uses this approach for some other algorithms. We could easily extend that to this case to keep the same standards. What do you think?

Reply:
I prefer the first option. Given that this is a corner case and should be addressed by the user. On the other hand, making `classes` a required parameter has the downside of limiting the application to "emerging classes".

Reply:
I see your point. So, the idea is to check whether the learner belongs to either sklearn or skmultiflow, and in case none of them check, we raise a Warning. Do you have any specific kind of warning type in mind?

Reply:
Another option is to raise an error in case the learner does not comply with either `sklearn` or `skmultiflow` standards. Thanks for suggesting that, @jacobmontiel. This last alternative is available in commit [97fe7dc](https://github.com/scikit-multiflow/scikit-multiflow/pull/223/commits/97fe7dc3c1b62c95018ca7f0bbd597b7360b34c8).

Reply:
Seems good to me. Thanks!